### ISSUE TITLE
Hide border looks a bit off when using a GitHub dark extension or a stylus

### ISSUE THREAD

**Issue by Anish-Shobith** [anuraghazra/github-readme-stats#159]
Hello there, 

So I was trying some cool stuff today and I noticed that when you have hide_border turned off (by default), it adds a a small stroke of white line around the corner which looks a bit odd when you are using a [GitHub dark theme](https://github.com/StylishThemes/GitHub-Dark) (I use [stylus](https://chrome.google.com/webstore/detail/stylus/clngdbkpkpeebahjckkjfobafhncgmne)) , Here is a hide_border enabled and disabled 
![image](https://user-images.githubusercontent.com/38991749/88284250-bb105c00-cd0a-11ea-8e85-e237d5cef092.png)
If you see that the enabled one at the top the text is not readable , I know that the background color can be modified and changed according to users will , and I did so , but this requires hide_border field to be disabled (default) but this creates a issue for users who are using a dark theme (shown below)
```md
<img src="https://github-readme-stats.vercel.app/api?username=Anish-Shobith&show_icons=true&icon_color=79ff97&text_color=9f9f9f&bg_color=151515">
```
![image](https://user-images.githubusercontent.com/38991749/88285570-f9a71600-cd0c-11ea-896f-cdb06be7fe4a.png)

If you see it adds a white border across the image, If i enable the hide_border option then it makes it unreadable for people using a normal theme (shown below)
```md
<img src="https://github-readme-stats.vercel.app/api?username=Anish-Shobith&hide_border=true&show_icons=true&icon_color=79ff97&text_color=9f9f9f&bg_color=151515">
```
![image](https://user-images.githubusercontent.com/38991749/88285713-35da7680-cd0d-11ea-9f0e-be7cb92c7394.png)

I have attached the code too with the image , I hope that helps during testing , and even hyper linked the stylus for chrome browser but its available even for other browsers and even the GitHub Dark theme I use.

Additional Information:
Browser : 
![image](https://user-images.githubusercontent.com/38991749/88285927-894cc480-cd0d-11ea-9a3f-16a1ebdb9f66.png)
GitHub Dark Config: 
![image](https://user-images.githubusercontent.com/38991749/88286055-ca44d900-cd0d-11ea-86ed-055f224a9204.png)

Wish I have provided enough information, If not please feel free to mention me.

Possible solution: 
I think that hiding border also removes the background color, which is causing this issue , if there was a option to hide only the border part or only the background part it could be nice 

And finally thanks for this wonderful project you created, It has helped so many developers including me ❤️.

**Comment by anuraghazra** [comment]
@Anish-Shobith Thanks for opening the issue :D

Yes it does have a white border around the card and we are exactly fixing the issue you mentioned, check out #151

**Comment by Anish-Shobith** [comment]
> Yes it does have a white border around the card and we are exactly fixing the issue you mentioned, check out #151
Sweet!

**Comment by terror** [comment]
@Anish-Shobith just checked this out, I am working on a fix over on my deployment: https://github-readme-stats-7nscindis.vercel.app/

Feel free to test it out and let me know what you think

Preview:
<img src="https://github-readme-stats-7nscindis.vercel.app/api?username=Anish-Shobith&show_icons=true&icon_color=79ff97&text_color=9f9f9f&bg_color=151515&hide_border=true"/>

**Comment by anuraghazra** [comment]
Fixed in #151

**Comment by Anish-Shobith** [comment]
@terror , Thank you 👍
